src/components/Chat/BottomSheetGif.js

```diff
@@ -10,6 +10,7 @@ import {
   FlatList
 } from 'react-native';
 import FastImage from 'react-native-fast-image';
+import {debounce} from 'lodash';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The lodash library is being imported in its entirety to use just the `debounce` function. This can lead to unnecessary increase in bundle size. Consider importing only the required function.
```diff
- import {debounce} from 'lodash';
+ import debounce from 'lodash/debounce';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->